test/modules/vass/use-depth.good

```diff
@@ -1,2 +1,6 @@
 use-depth.chpl:21: error: symbol x is multiply defined
 use-depth.chpl:37: note: also defined here
+use-depth.chpl:22: error: symbol y is multiply defined
+use-depth.chpl:28: note: also defined here
```

Comment:
I don't think this is worth changing in this PR (unless it's easy), but it seems backwards to me that the symbol obtained through the `use` is listed as the "multiply defined" case rather than the local variable given that it's the scope where the local variable is defined that caused the problem.  Put another way, if the module was in a library somewhere, it would seem odd that the primary error message was saying "there's an error in this standard library (because note that you also defined this symbol in your code)", whereas I think the bug is clearly in your code (for using rather than importing the library and then defining a symbol of that name), so the error message should point there and the note to the library rather than vice-versa.

I do think it would be worth filing a future and issue against this, though.

Reply:
Argh, just noticed that I'd noted this earlier a bit further down in the file and that it was resolved, sorry about that.  I do think a future would be a better way of not losing sight of this than a private issue.

Reply:
FWIW, I think this would involve a lot of logging to do well - in order to know "the use is the one causing the problem", we have to keep track of the steps we used to finally get the symbol.  It's harder to recreate those steps when we have two symbols that conflict if we weren't already logging it, and in order to log it from the get-go we'd have to have a stack of the uses and imports we've followed to get to a certain point and keep that attached to a symbol, all for improving an error message that is not needed for the vast majority of symbol resolution

Reply:
Oh, is that right?  I as imagining that we'd just take the AST nodes passed to the USR_FATAL_CONT and USR_NOTE and swap them to get the fix.

Reply:
Oh, I think I misinterpreted, never mind